app/services/github_client/repository.rb

```diff
@@ -15,7 +15,7 @@ def pull_requests
         request['state'] = 'all'
       end
       response_body = response.body
-      response_body.empty? ? [] : JSON.parse(response_body, symbolize_names: true)
+      JSON.parse(response_body, symbolize_names: true) unless response_body.empty?
```

Comment:
I'm sorry, but I'll have to ask again: Is there any scenario where the response body will be empty?